### ISSUE TITLE
Internal Facilities Grid

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1703]
#### Description:
As an internal user, I can see the facilities for all operators

#### Acceptance Criteria:

Given I am on the dashboard 
When I click on the facilities tile
Then I am brought to the facilities grid where I can see all facilities of all operators
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7861-362&t=lP4wgjHuBjopAaSN-4)

Given I am on the operation details page for an operation
When I click on the facilities tile
Then I am brought to the facilities grid and the operator and operation column is pre-filtered to show only that operation's facilities 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by nanyangpro** [comment]
Design/Prototype links are added or updated with AC refined further.

**Comment by andrea-williams** [comment]
to be decided later - is this a feature that's needed at all?

**Comment by patriciarussellCAS** [comment]
@zoeyli-46 something to consider when talking to IRC about internal views. Can they see all the facilities in Reporting? Would it be more useful there or in Admin? Let's explore this requirement/need a bit more!
